### PR TITLE
Release Operator 2.6.0

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: docs-source/content/_index.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ using the operator to deploy and run a WebLogic domain container-packaged web ap
 ***
 #### Current production release
 
-The [current production release of the operator](https://github.com/oracle/weblogic-kubernetes-operator/releases) is 2.5.0.
-This release was published on February 26, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
+The [current production release of the operator](https://github.com/oracle/weblogic-kubernetes-operator/releases) is 2.6.0.
+This release was published on June 1, 2020. See the operator prerequisites and supported environments [here]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
```
Comment (rjeberhard) [inline docs-source/content/_index.md:27]: I'll fix this date.

File: docs-source/content/samples/simple/domains/fmw-domain/_index.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ The content of the generated `domain.yaml`:
 #
 # This is an example of how to define a Domain resource.
 #
-apiVersion: "weblogic.oracle/v4"
+apiVersion: "weblogic.oracle/v5"
```
Comment (rjeberhard) [inline docs-source/content/samples/simple/domains/fmw-domain/_index.md:171]: Needs to be v7.

File: docs-source/content/release-notes.md [hunk 3]
```diff
@@ -8,8 +8,9 @@ draft: false
...
-| February 26, 2020 | v2.5.0 | no | Support for Helm 3.x and OpenShift 4.3.  Operator can be installed in a namespace-dedicated mode where operator requires no cluster-level Kubernetes privileges. This version is not supported on Kubernetes 1.16+, check the [prerequisites]({{< relref "/userguide/introduction/introduction#operator-prerequisites" >}}).
+| June 1, 2020 | v2.6.0 | no | Kubernetes 1.16, 1.17, and 1.18 support. Removal of support for Kubernetes 1.13 and earlier. This release can be run in the same cluster with operators of either 2.5.0 and below, or with 3.x providing an upgrade path. Certified support of Oracle Linux Cloud Native Environment (OLCNE) 1.1 with Kubernetes 1.17.0.
```
Comment (rosemarymarano) [inline docs-source/content/release-notes.md:11]: And, you'll fix this date, too?

File: docs/domains/Domain.md [hunk 4]
```diff
@@ -20,16 +20,16 @@ DomainSpec is a description of a domain.
...
+| `image` | string | The WebLogic Docker image; required when domainHomeInImage is true; otherwise, defaults to container-registry.oracle.com/middleware/weblogic:12.2.1.4. |
 | `imagePullPolicy` | string | The image pull policy for the WebLogic Docker image. Legal values are Always, Never and IfNotPresent. Defaults to Always if image ends in :latest, IfNotPresent otherwise. |
 | `imagePullSecrets` | array of [Local Object Reference](k8s1.13.5.md#local-object-reference) | A list of image pull secrets for the WebLogic Docker image. |
-| `includeServerOutInPodLog` | Boolean | If true (the default), the server .out file will be included in the pod's stdout. |
-| `logHome` | string | The in-pod name of the directory in which to store the domain, node manager, server logs, and server  *.out files |
+| `includeServerOutInPodLog` | Boolean | If true (the default), then the server .out file will be included in the pod's stdout. |
+| `logHome` | string | The in-pod name of the directory in which to store the domain, Node Manager, server logs, and server *.out files |
```
Comment (rosemarymarano) [inline docs/domains/Domain.md:32]: Period at the end of the sentence, after files.

File: docs/domains/Domain.md [hunk 5]
```diff
@@ -83,11 +83,11 @@ An element representing a cluster in the domain configuration.
 | `serverStartPolicy` | string | The strategy for deciding whether to start a server. Legal values are NEVER, or IF_NEEDED. |
 | `serverStartState` | string | The state in which the server is to be started. Use ADMIN if server should start in the admin state. Defaults to RUNNING. |
 
-### Experimental
+### Configuration
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `istio` | [Istio](#istio) | Istio service mesh integration configuration. |
+| `istio` | [Istio](#istio) | The Istio service mesh integration settings |
```
Comment (rosemarymarano) [inline docs/domains/Domain.md:90]: Period at the end of the sentence, after settings.

**Comment by rjeberhard** [comment]
After correcting Rosemary's issues, I'm going to close this PR, get the changes to the `release/2.6.0` branch and make a PR from there.

**Pull request closed without merging**